src/components/OSCALAnchorLinkHeader.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import Stack from "@mui/material/Stack";
+import LinkIcon from "@mui/icons-material/Link";
+import { Link } from "react-router-dom";
+import Fade from "@mui/material/Fade";
+
+export default function OSCALAnchorLinkHeader(props) {
+  const [hover, setHover] = useState(false);
+  const onEnter = () => {
+    setHover(true);
+  };
+  const onLeave = () => {
+    setHover(false);
+  };
+
+  const editString = (str) => str.replaceAll(" ", "-").toLowerCase();
+
+  return (
+    <Stack
+      direction="row"
+      gap={1}
+      alignItems="start"
+      onMouseEnter={onEnter}
+      onMouseLeave={onLeave}
+      id={editString(props.title)}
+    >
+      {props.title}
+      {!hover ? (
+        ""
+      ) : (
+        <Link to={{ hash: `#${editString(props.title)}` }}>
+          <Fade in={hover}>
+            <LinkIcon />
+          </Fade>
+        </Link>
+      )}
```

Comment:
Can you clarify for me how this is different from

```suggestion
      {!hover && (
        <Link to={{ hash: `#${editString(props.title)}` }}>
          <Fade in={hover}>
            <LinkIcon />
          </Fade>
        </Link>
      )}
```

Reply:
There really isn't much difference, what you're suggesting is much cleaner. Originally to make sure that there was appearing after the header text, I had some test text, and ended up just making it an empty string. The ternary operator isn't needed anymore. Will make the change.